dev/preview/workflow/vm/template/gcr-pull-secret-job.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: gcr-refresh-token
+spec:
+  schedule: "30 * * * *"
+  successfulJobsHistoryLimit: 1
+  suspend: false
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 60
+      template:
+        spec:
+          serviceAccountName: gcr-refresh-token
+          containers:
+          - name: gcr-refresh-token
+            image: chainguard/kubectl:latest-dev
+            command:
+              - /bin/sh
+              - -c
+              - |-
+                ACCOUNTS=$(wget -q -O - --header "Metadata-Flavor: Google" "http://169.254.169.254/computeMetadata/v1/instance/service-accounts/" | tr -d '\r')
+                NON_DEFAULT_ACCOUNTS=$(echo "$ACCOUNTS" | grep -v "^default$")
+                FIRST_NON_DEFAULT_ACCOUNT=$(echo "$NON_DEFAULT_ACCOUNTS" | head -1)
+                TOKEN=$(wget -q -O - --header "Metadata-Flavor: Google" "http://169.254.169.254/computeMetadata/v1/instance/service-accounts/$FIRST_NON_DEFAULT_ACCOUNT/token")
+                ACCESS_TOKEN=$(echo "$TOKEN" | grep -o '"access_token": *"[^"]*"' | sed 's/"access_token": *"\([^"]*\)"/\1/')
+
+                AUTH_TOKEN=$(echo -n _dcgcloud_token:${ACCESS_TOKEN} | base64 -w0)
```

Comment:
`_dcgcloud_token` appears special. What is it for? Short lived tokens? It would be helpful to leave a comment or URL for reference.